cms/static/sass/edx-icons.scss

```diff
@@ -3,13 +3,8 @@
 
 // Table of Contents
 // * +Libs and Resets
-// * +edx-icons
```

Comment:
Can you not just delete this entire file? It's named "edx-icons", so I don't think it is related to anything else.